pmd-ant/src/test/resources/net/sourceforge/pmd/ant/xml/pmdtasktest.xml

```diff
@@ -27,7 +27,8 @@
 
     <target name="testWithShortFilenames">
         <pmd noCache="true" shortFilenames="true">
-            <ruleset>${pmd.home}/src/test/resources/rulesets/dummy/basic.xml</ruleset>
+            <!-- note: this ruleset is in pmd-core/src/test/resources -->
+            <ruleset>rulesets/dummy/basic.xml</ruleset>
```

Comment:
Nice! :+1: (Just looked up how this works: pmd-core creates a test-jar, and pmd-ant has an additional test-scoped dependency on the pmd-core's test-jar)